src/main/window-lifecycle.js

```diff
@@ -55,6 +55,7 @@ const defaultWindowOptions = {
     devTools: __DEV__ || DEV_TOOLS,
     experimentalFeatures: true,
     nodeIntegration: true,
+    contextIsolation: false,
```

Comment:
unfortunately

Reply:
Making `contextIsolation` false breaks the security model if we import external scripts from what I understand. Gab pointed out in the context of the chatbot poc https://ledgerhq.atlassian.net/browse/LL-2349 that without it scripts could go rogue on our app, I'm scared of this change 😕

Reply:
Its either that or we wait for upgrade to deactivate nodeIntegration that requires this :disappointed:

Reply:
I lack the knowledge of whether this is dangerous or not so I'd rather not approve